pkg/sql/exec_util.go

```diff
@@ -1057,6 +1057,20 @@ func golangFillQueryArguments(args ...interface{}) (tree.Datums, error) {
 				switch {
 				case val.IsNil():
 					d = tree.DNull
+				case val.Type().Elem().Kind() == reflect.String:
+					a := tree.NewDArray(types.String)
+					for i := 0; i < val.Len(); i++ {
```

Comment:
looks like `i` is already used as the outer loop variable. so we need a different name here

Reply:
Ah, yes, noticed that in the TeamCity failure — resolved